MK_solutions_week_2

```diff
@@ -0,0 +1,134 @@
+Select all that apply:
+```
+[X] Strings
+[X] Booleans
+[X] Undefined
+[] NaN
+[] Integers // By this are you referring to "numbers" datatypew?
+[] Arrays
+[X] Null
+```
+
+## Question #2
+
+Explain what is a REPL, and why is it important for us as developers and help with debugging?
+
+```
+REPL, or read-eval-print loop is a set of expressions that parse, evaluate data structures and then print out the result to the user. It is beneficial for debugging because it allows us as developers see the result when printed. The REPL environment also allows us to handle the error when they occur.
+```
+### Question #3
+
+**Given the Following Array**
+
+var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
+
+Create a For Loop that outputs the following string for each piece of fruit in the console. "I want to eat a [fruit]"
+
+```
+for (i = 0; i < food.length; i++) {
+  var text = " ";
+  text += "I want to eat a [fruit]";
+}
```

Comment:
Close.  Please try this in javascript console or repl.it.  Does it do what you expect?